internal/testhelpers/srvstatus/srvstatus.go

```diff
@@ -0,0 +1,53 @@
+package srvstatus
```

Comment:
This should be in a test file

Reply:
Just tried and the retryhttp test can't import after: "no non-test Go files in srvstatus"